README.md

```diff
@@ -65,6 +65,10 @@ Amp\Loop::run(function () {
 php example.php
 ```
 
+## Contributing
+
+Please read [CONTRIBUTING.md](https://github.com/amphp/amp/blob/master/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
```

Comment:
Could you add code tags around `CONTRIBUTING.md`, please?

Reply:
Done